attachment_log/models/ir_attachment.py

```diff
@@ -0,0 +1,52 @@
+from odoo import _, models
+from odoo.fields import Datetime
+
+
+class IrAttachment(models.Model):
+    _inherit = "ir.attachment"
+
+    def _send_attachment_notification(self, is_create=False, is_unlink=False):
+        if is_create:
+            action = _("attached")
+        elif is_unlink:
+            action = _("unlinked")
+        else:
+            return
+        record = self.env[self.res_model].sudo().browse(self.res_id)
+        message_text = _(
+            "<b>%(username)s</b> %(action)s a file: <i>%(filename)s</i>"
+        ) % {
+            "username": self.create_uid.name,
+            "filename": self.name,
+            "action": action,
+        }
+        message = record.message_post(
```

Comment:
You should use a custom message subtype for such messages in order to allow more flexible configuration.